### ISSUE TITLE
Non-backwards-compatible version released to crates.io with only minor version bump

### ISSUE THREAD

**Issue by jbg** [antifuchs/ratelimit_meter#27]
Thanks so much for this crate and ratelimit_futures, they've been super useful to me.

The recent 4.1.0 release of ratelimit_meter seems to be backward-incompatible with 4.0.x, since ratelimit_futures no longer builds:

```
error[E0107]: wrong number of type arguments: expected 1, found 0
  --> src/lib.rs:65:41
   |
65 |     <A as Algorithm>::NegativeDecision: NonConformance,
   |                                         ^^^^^^^^^^^^^^ expected 1 type argument

error[E0107]: wrong number of type arguments: expected 1, found 0
  --> src/lib.rs:74:41
   |
74 |     <A as Algorithm>::NegativeDecision: NonConformance,
   |                                         ^^^^^^^^^^^^^^ expected 1 type argument

error[E0107]: wrong number of type arguments: expected 1, found 0
   --> src/lib.rs:101:41
    |
101 |     <A as Algorithm>::NegativeDecision: NonConformance,
    |                                         ^^^^^^^^^^^^^^ expected 1 type argument
```

... but only the minor version was bumped, so the ratelimit_futures crate (and anything else depending on ratelimit_meter using a normal "caret" dependency) automatically pulls the newer version and breaks.

**Comment by antifuchs** [comment]
Eep, thanks for letting me know - I really should pull both crates into the same repo, probably to reduce the likelihood of one breaking without the other noticing.

I'll adjust the dependency in ratelimit_futures & fix the incompatible signatures there.

(Also, thanks so much for the kind words - I'm extremely happy you're using this humble project for something useful!!)

**Comment by antifuchs** [comment]
Bit over-eager, bors bot! But I just released v4.1.1, which should restore backwards-compatibility & make ratelimit_futures build again (:
